johns-hopkins-format_Italy.R

```diff
@@ -22,7 +25,7 @@ x <- x %>%
     Long = "long"
   ) %>%
   mutate(
-    data = format(data, format = "%m/%d/%y")
+    data = str_trim(glue("{month(data)}/{day(data)}/{substr(year(data), 3, 4)}"))
```

Comment:
Without `str_trim`, dates after 3/1 were coming with an extra space at the end -- no idea what that was all about, but `str_trim` fixes it.